xonsh/hist.py

```diff
@@ -0,0 +1,42 @@
+"""Implements the xonsh history object"""
+
+from collections import OrderedDict
+import json
+import time
+
+class History(object):
+
+    ordered_history = OrderedDict()
+
+    def load_history(self, hist_file="~/.xonsh_history.json"):
+        """Loads previous history from ~/.xonsh_history.json if it exists.
+
+        Parameters
+        ----------
+        hist_file : str, optional
+            Path of the history file.
+
+        Returns
+        -------
+            None
```

Comment:
Not that this blocks, but you probably don't need to document that a function or method returns None :)